Dedicated memory overhead clarification (#17448)
cockroachdb/docs: src/current/cockroachcloud/plan-your-cluster.md

A single node cluster is only appropriate for single-region application development and testing. For single-region production deployments, we recommend a minimum of 3 nodes. The number of nodes you choose also affects your storage capacity and performance. See the [Example](#dedicated-example) for more information.

> Note that {{ site.data.products.dedicated }} clusters use some provisioned RAM for system overhead factors such as filesystem cache and sidecars, so the full amount of memory may not be available to the cluster.

Reviewer: ```suggestion
A {{ site.data.products.dedicated }} cluster uses some of its provisioned RAM for the node's operating system and system overhead factors such as filesystem cache and sidecars, so the full amount of memory may not be available to the cluster's workloads.
```
